src/scaladoc/scala/tools/nsc/doc/html/SyntaxHigh.scala

```diff
@@ -145,7 +145,23 @@ private[html] object SyntaxHigh {
     }
 
     def strlit(i: Int): String = {
-      val out = new StringBuilder("\"")
+      val out = new StringBuilder()
+
+      def rawstrlit0(i: Int, bslash: Boolean): Int = {
```

Comment:
The result `Int` of that method is discarded, so let's change the return type to `Unit`. While you're at it, also the result of method `strlit0` is unused.